CHANGELOG.rst

```diff
@@ -0,0 +1,36 @@
+Change Log
+==========
+
+..
+   This file loosely adheres to the structure of http://keepachangelog.com/ ,
+   but in reStructuredText instead of Markdown.
+
+   This project adheres to Semantic Versioning (http://semver.org/).
```

Comment:
```suggestion
   This project adheres to Semantic Versioning (https://semver.org/).
```

Reply:
It's funny, SemVer has two major versions, but no changelog to tell you which one to use. But I don't think it matters, because we're not likely to follow it precisely anyhow.

Reply:
@timmc-edx Can you say more about "not likely to follow it precisely"?

Reply:
(Specifically, I'm not sure that its mechanism for prereleases and whatnot matches Python conventions.)

Reply:
Python is pretty flexible about version numbers: https://www.python.org/dev/peps/pep-0440/

Reply:
Yeah, I've seen libraries that just use the date. :-P

The difference I'm thinking about is "we're going to follow the semver.org v2.0.0 spec" vs. "we're going to follow semver as it is broadly understood, and here's a link if you don't know what that means". I get the sense that most people who "follow semver" are just following it in the spirit of "version numbers indicate API stuff" with major version bumps for breaking changes. (Maybe that's just projection, though?)

I'm OK with either way.

Reply:
Once we land the OEP on Semantic Versioning, I'd like to point there instead of the spec.  See https://github.com/edx/open-edx-proposals/pull/149
If you have comments around versioning, feel free to add them to the OEP PR.